packages/tools/src/launchEditor.ts

```diff
@@ -114,9 +141,21 @@ function guessEditor() {
           return [COMMON_EDITORS[processName]];
         }
       }
-    } catch (error) {
-      // Ignore...
+    } else if (process.platform === 'win32') {
+      const output = execSync(
+        'tasklist /NH /FO CSV /FI "SESSIONNAME ne Services"',
+      ).toString();
+
+      const processNames = Object.keys(COMMON_WINDOWS_EDITORS);
+      for (let i = 0; i < processNames.length; i++) {
+        const processName = processNames[i];
+        if (output.indexOf(processName) !== -1) {
```

Comment:
Nit: "output.includes" instead of indexof

Though this would also seemingly catch substring. Can we split the task list and look for an exact match?

Reply:
I was using the same logic as the macOS version.  But yes, I can go ahead and make the windows logic more correct.